tests/unit/test_search_results1.py

```diff
@@ -0,0 +1,135 @@
+"""Module to test the KytosGraph in graph.py."""
+# Core modules to import
+from kytos.core.link import Link
+
+# module under test
+from tests.unit.test_search_results import TestSearchResults
+
+
+class TestSearchResults1(TestSearchResults):
+    """Tests for the graph class."""
+
+    def test_path1(self):
+        """Tests a simple, definetly possible path"""
+        self.setup()
```

Comment:
This should be automatically called by changing the method's name.

Reply:
Fixed, thank you!